src/components/forms/LoginForm.jsx

```diff
@@ -0,0 +1,45 @@
+
+import { useForm } from 'react-hook-form';
+
+export default function LoginForm({ onSubmit }) {
+    const {
+        register,
+        handleSubmit,
+        getValues,
+        formState: { errors },
+    } = useForm();
+
+
+    return (
+        <form onSubmit={handleSubmit(onSubmit)}>
+            <input {...register('email', {
+                required: "Email is required",
+            })}
+                type='email'
+                placeholder='Email'
+            />
+            {errors.email && (<p>{`${errors.email.message}`}</p>)}
+            <input {...register('password', {
+                required: "Password is required",
+                minLength: {
+                    value: 10,
+                    message: "Password must be at leat 10 characters"
+                }
+            })}
+                type='password'
+                placeholder='Password'
+            />
+            {errors.password && (<p>{`${errors.password.message}`}</p>)}
+            <input {...register('confirmPassword', {
+                required: "Confirm password is required",
+                validate: (value) =>
+                    value === getValues("password") || "Passwords must match"
+            })}
+                type="password"
+                placeholder='Confirm password'
+            />
+            {errors.confirmPassword && (<p>{`${errors.confirmPassword.message}`}</p>)}
+            <input type="submit" />
```

Comment:
submit should be a button, I understand if you prefer to leave it to the fe team (:

Reply:
Honestly didn't noticed this :) I followed tutorial for react-hook-form :)